packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -51,8 +51,8 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
       viewport,
       selectionMode,
       loadingText,
-      isSelected,
-      selected,
+      isSelected = false,
+      selected = false,
```

Comment:
With these defaults, `currentlySelected` below is now guaranteed to be defined, so we could probably change line 102 from 

```tsx
aria-checked={!!currentlySelected}
```

to 

```tsx
aria-checked={currentlySelected}
```